examples/apps/data-object-grid/package.json

```diff
@@ -36,7 +37,6 @@
     "test": "npm run test:jest",
     "test:jest": "jest",
     "test:jest:verbose": "cross-env FLUID_TEST_VERBOSE=1 jest",
-    "tsc": "tsc",
```

Comment:
I'm assuming you've checked that fluid-build is okay with removing this script? I know it at least wants this to exist if the package has a `build:docs` script, but I don't know if there are other cases where this matters.

Reply:
I tested these changes with fluid build (via build fast). I did not check that that no new warnings are produced because we have almost a thousands of them already and so its hard to check. When I made https://github.com/microsoft/FluidFramework/pull/10845 I did make sure it was warning free though and this is similar.